vars/githubscm.groovy

```diff
@@ -11,44 +11,44 @@ def resolveRepository(String repository, String author, String branches, boolean
             targets: [branches])
 }
 
-def checkoutIfExists(String repository, String author, String branches, String defaultAuthor, String defaultBranches, boolean mergeTarget = false, String credentialsId = 'kie-ci1-token') {
+def checkoutIfExists(String repository, String author, String branches, String defaultAuthor, String defaultBranches, boolean mergeTarget = false, String tokenCredentialsId = 'kie-ci1-token', String usernamePasswordCredentialsID = 'kie-ci') {
```

Comment:
too many default arguments. What about to replace those two by one map.
`def credentials['tokenCredentialsId' : 'kie-ci1-token', 'usernamePasswordCredentialsID' = 'kie-ci']` ?

Reply:
Changed =D